### ISSUE TITLE
Can't resolve @vercel/edge-config peer dependency

### ISSUE THREAD

**Issue by mattiasw** [splitio/vercel-integration-utils#20]
`@vercel/edge-config` has gotten a major version update with breaking changes. When I try to install it I get this error:

```
npm warn Found: @vercel/edge-config@0.4.1
npm warn node_modules/@vercel/edge-config
npm warn   peer @vercel/edge-config@">=0.1.0 <1.0.0" from @splitsoftware/vercel-integration-utils@1.1.1
npm warn   node_modules/@splitsoftware/vercel-integration-utils
npm warn     @splitsoftware/vercel-integration-utils@"^1.1.1" from the root project
npm warn   1 more (the root project)
npm warn
npm warn Could not resolve dependency:
npm warn peer @vercel/edge-config@">=0.1.0 <1.0.0" from @splitsoftware/vercel-integration-utils@1.1.1
npm warn node_modules/@splitsoftware/vercel-integration-utils
npm warn   @splitsoftware/vercel-integration-utils@"^1.1.1" from the root project
```

Maybe it works just fine anyway but it would be nice to avoid the warning and make sure they actually work well together.

Here are the breaking changes:
https://github.com/vercel/storage/releases/tag/%40vercel%2Fedge-config%401.0.0

**Comment by EmilianoSanchez** [comment]
Hi @mattiasw 

We have released v1.2.0 of the `@splitsoftware/vercel-integration-utils` package, which extends the `@vercel/edge-config` peer dependency to support version 1.0.0 and later. We have also validated its compatibility with the latest Edge Config client and that no breaking changes affect the package implementation.

I’ll close this issue for now. Thank you for bringing this issue to our attention!
